templates/front-page.html

```diff
@@ -11,6 +11,9 @@
 	"className": "site-main",
 	"layout": {
 		"type": "default"
+	},
+	"metadata":{
+		"name":"Site Main"
```

Comment:
One nitpicky thing - I had included spaces after `:` throughout the site. Both work, and I don't feel strongly about my method, but I do think it should be consistent. Any reason why you think we should do spaces vs no spaces?
```suggestion
	"metadata": {
		"name": "Site Main"
```

Reply:
@kaitlinbolling Ah good catch, no strong opinions regarding spaces vs no spaces but definitely agree it should be consistent. I made that updated in [e6e2d21](https://github.com/alleyinteractive/create-wordpress-theme/pull/22/commits/e6e2d2100a9c86657115409d8d41c0b8d59abc88) Just out of curiosity, are you running any sort of formatting tool to maintain consistency? 

Along the same lines, something that might be worth considering as we build the templates is when using the Site Editor to build out blocks and then using the export tool to eject the theme all the formatting is lost. No strong preference on this but maybe if we are going to be leaning in on the building blocks in browser workflow we may want to stick with the exported format to avoid the additional overhead of maintaining the formatting?

Reply:
I am not using any kind of formatter - but if one was available that would be awesome. And good point about copying from the site editor. I did that a lot and it's a pain to format it, but it's so much more readable so I'm torn!